source/openapi-admin-v3.yaml

```diff
@@ -5632,6 +5771,157 @@ components:
+            string. You can use this identifier when configuring permissions.
+        created:
+          type: string
+          description: |-
+            A string representing the timestamp at which the Edge Server 
+            instance was created.
+        name:
+          type: string
+          description: |-
+            The human-readable name supplied when 
+            [creating the Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/manage-edge-servers/#create-an-edge-server-instance) 
+            instance.
+        user_id:
+          type: string
+          description: |-
+            The Edge Server itself is a user. This is the unique identifier of 
+            the Edge Server user, of type ObjectId, as a string.
+        status:
+          type: string
+          enum:
+            - "active"
+            - "deleted"
+            - "failed"
+            - "registered"
+            - "unregistered"
+          description: |-
+            The current status of the Edge Server instance.
+        hostname:
+          type: string
+          description: The public-addressable DNS of the Edge Server instance.
+        last_connection:
+          type: string
+          description: |-
+            A string representing the timestamp at which the Edge Server 
+            instance last connected to Atlas.
+      example:
+        id: 6622b8f5118368b943d228a9
+        created: 2024-04-19T08:51:00Z
+        name: Edge-Store-42
+        user_id: 6622b8f5118368b943d2288a
+        status: active
+        hostname: ec2-13-58-70-88.us-east-2.compute.amazonaws.com
+        last_connection: 2024-04-19T09:51:00Z
+    EdgeServerInfo:
+      type: object
+      properties:
+        id:
+          type: string
+          description: |-
+            A unique identifier for the Edge Server, of type ObjectID, as a 
+            string. You can use this identifier when configuring permissions.
+        created:
+          type: string
+          description: |-
+            A string representing the timestamp at which the Edge Server 
+            instance was created.
+        name:
+          type: string
+          description: |-
+            The human-readable name supplied when 
+            [creating the Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/manage-edge-servers/#create-an-edge-server-instance) 
+            instance.
+        user_id:
+          type: string
+          description: |-
+            The Edge Server itself is a user. This is the unique identifier of 
+            the Edge Server user, of type ObjectId, as a string.
+        status:
+          type: string
+          enum:
+            - "active"
+            - "deleted"
+            - "failed"
+            - "registered"
+            - "unregistered"
+          description: |-
+            The current status of the Edge Server instance.
+        hostname:
```

Comment:
this is not going to be populated for the foreseeable future; we should just omit it